lib/rules/declaration-property-value-no-unknown/index.mjs

```diff
@@ -64,16 +64,30 @@ const rule = (primary, secondaryOptions) => {
 		const propertiesSyntax = {
 			overflow: '| overlay', // csstree/csstree#248
 			width: '| min-intrinsic | -moz-min-content | -moz-available | -webkit-fill-available', // csstree/csstree#242
+			'anchor-name': 'none | <custom-property-name>#',
 			'field-sizing': 'content | fixed',
 			'text-box-edge':
 				'auto | [ text | cap | ex | ideographic | ideographic-ink ] [ text | alphabetic | ideographic | ideographic-ink ]?',
 			'text-box-trim': 'none | trim-start | trim-end | trim-both',
 			'text-spacing-trim': 'normal | space-all | space-first | trim-start',
 			'text-wrap-style': 'auto | balance | pretty | stable',
+			'view-timeline-axis': '[ block | inline | x | y ]#',
+			'view-timeline-inset': '[ [ auto | <length-percentage> ]{1,2} ]#',
+			'view-timeline-name': '[ none | <custom-property-name> ]#',
+			'view-timeline':
+				"[ <'view-timeline-name'> [ <'view-timeline-axis'> || <'view-timeline-inset'> ]? ]#",
+			// <custom-ident> represents any valid CSS identifier that would not be misinterpreted as a pre-defined keyword in that property’s value definition
+			// i.e. reserved keywords don't have to excluded
+			// w3c/csswg-drafts#9895
+			'view-transition-name': 'none | <custom-ident>',
```

Comment:
I couldn't find any way to exclude a keyword.
i.e. something that could merge `<custom-ident>` with `auto#{0}`